Content.Server/Access/Systems/AccessOverriderSystem.cs

```diff
@@ -242,10 +242,10 @@ private void TryWriteToTargetAccessReaderId(EntityUid uid,
         var removedTags = oldTags.Except(newAccessList).Select(tag => "-" + tag).ToList();
 
         _adminLogger.Add(LogType.Action, LogImpact.Medium,
-            $"{ToPrettyString(player):player} has modified {ToPrettyString(component.TargetAccessReaderId):entity} with the following allowed access level holders: [{string.Join(", ", addedTags.Union(removedTags))}] [{string.Join(", ", newAccessList)}]");
+            $"{ToPrettyString(player):player} has modified {ToPrettyString(accessReaderEnt.Value):entity} with the following allowed access level holders: [{string.Join(", ", addedTags.Union(removedTags))}] [{string.Join(", ", newAccessList)}]");
 
-        accessReader.AccessLists = ConvertAccessListToHashSet(newAccessList);
-        Dirty(component.TargetAccessReaderId, accessReader);
+        accessReaderEnt.Value.Comp.AccessLists = ConvertAccessListToHashSet(newAccessList);
+        Dirty(accessReaderEnt.Value, accessReaderEnt.Value.Comp);
```

Comment:
```suggestion
        Dirty(accessReaderEnt.Value);
```
Nitpick. I'm pretty sure you can just pass the Entity<T> to Dirty instead of breaking it up.

Reply:
I think it needs a `Dirty<AccessReaderComponent>(accessReaderEnt.Value)` in that case.

Reply:
Rider says its redundant, is this really a good way to do it? I don't think ive seen anything dirtied this way before.

Reply:
`accessReaderEnt` is an `Entity<AccessReaderComponent>`
so only that component gets dirtied
it's just a more compact way of writing both the uid and component separately